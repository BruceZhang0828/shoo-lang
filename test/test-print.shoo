int a = 5;
println(str_of_int(a));
println(str_of_int(5));

// TODO need to implement str_of_float
/*float b = 1.234567;
println(str_of_float(b));
println(str_of_float(1.234567));*/

bool c = false;
println(str_of_bool(c));
println(str_of_bool(false));
bool d = true;
println(str_of_bool(d));
println(str_of_bool(true));

string e = "hello";
println(e);
println("hello");

string f = "hi";
print(f);
println("");
