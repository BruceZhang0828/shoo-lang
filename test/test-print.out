5
5
false
false
true
true
hello
hello
hi
